assets/base/i18n/en.json

```diff
@@ -10,6 +10,26 @@
   "about.teamEditiont0": "Team Edition",
   "about.teamEditiont1": "Enterprise Edition",
   "about.title": "About {appTitle}",
+  "accessibility.attachment_button": "attachment button",
+  "accessibility.channel_nav_bar": "channel nav bar",
+  "accessibility.channel_post_list": "channel post list screen",
+  "accessibility.login": "login screen",
+  "accessibility.login_options": "login options screen",
+  "accessibility.login_options.email_button": "email login option button",
+  "accessibility.login_options.gitlab_button": "gitlab login option button",
+  "accessibility.login_options.logo_image": "logo image",
+  "accessibility.login.email_or_username input": "",
+  "accessibility.login.forgot_password_button": "forgot password button",
+  "accessibility.login.logo_image": "logo image",
+  "accessibility.login.password_input": "password input",
+  "accessibility.login.sign_in_button": "sign in button",
+  "accessibility.post_textbox": "post textbox",
```

Comment:
what the difference between this and the one below

Reply:
This is the container where one of the child is the text input, others are button like for at-mention, slash command, etc... Will try to come up with better naming convention.